docs/file-format.rst

```diff
@@ -535,3 +535,36 @@ Path                               Type
 /trees/indexes/insertion_order     H5T_STD_U32LE
 /trees/indexes/removal_order       H5T_STD_U32LE
 ==============================     ==============
+
+***************
+Legacy Versions
+***************
+
+Tree sequence files written by older versions of msprime are not readable by
+newer versions of msprime. For major releases of msprime, :command:`msp
+upgrade` will convert older tree sequence files to the latest version.
+
+However many changes to the tree sequence format are not part of major
+releases. The table below gives these versions (contained in the root group
+attribute, ``format_version`` as a pair ``(major, minor)``).
+
+=======    =================
+Version    Commit Short Hash
+=======    =================
+9.0        e504abd
+8.0        299ddc9
+7.0        ca9c0c5
+6.0        6310725
+5.0        62659fb
+4.0        a586646
+3.2        8f44bed
+3.1        d69c059
+3.0        7befdcf
+2.1        a26a227
+2.0        7c507f3
+1.1        04722d8
+1.0        04722d8
```

Comment:
these failed because they're the same o_O

Reply:
this happened because the changes from 04722d8 to 2.0 were not done on separate branches and merged in